#!/usr/bin/env python
PACKAGE = "virtual_tracks"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("target_latitude",                 str_t,    0,        "Target_latitude:  ",        "48.784205")
gen.add("target_longitude",                str_t,    0,        "target_longitude: ",        "11.473232")

exit(gen.generate(PACKAGE, "virtual_tracks", "track_params"))
